src/core/quantum_gate.jl

```diff
@@ -1599,6 +1598,22 @@ P(\\phi) = \\begin{bmatrix}
 """
 phase_shift(phi, T::Type{<:Complex} = ComplexF64) = DiagonalOperator(T[1.0, exp(im * phi)])
 
+"""
+    rotation_z(lambda)
+
+Return the `DiagonalOperator` that applies a rotation of `z` .
```

Comment:
symmetric phase offset should be mentioned

Reply:
Same as above